MarketDataMessage2.capnp

```diff
@@ -0,0 +1,108 @@
+using Cxx = import "/capnp/c++.capnp";
+using import "Exchanges.capnp".Exchange;
+
+$Cxx.namespace("proto::marketdata");
+@0xb58df63b06ce5a55;
+
+
+#######################################################################################################
+#                   COMMON TYPES
+#######################################################################################################
+
+
+struct Header {
+    clientID @0 :UInt64;
+    senderCompID @1 :Text = "<UNDEFINED>";
+}
+
+
+enum Chanel {
+	ticker @0;          # ticker data
+    orderbook @1;       # level 2 orderbook data
+}
+
+
+enum Side {
+    undefined @0;
+    buy  @1;
+    sell @2;
+}
+
+
+#######################################################################################################
+#                   MESSAGE
+#######################################################################################################
+
+
+struct MarketDataMessage2 {
```

Comment:
[INQUIRY]
What is the reason of the digit '2' in the end?

Reply:
Because we have old version of MarketDataMessage, which is still in use and will be used in parallel with this version. We can actually remove 2 from message type and keep it in file only.